python/servo/build_commands.py

```diff
@@ -764,6 +772,34 @@ def gstreamer_root(target, env):
         return None
 
 
+def angle_root(target):
+    arch = {
+
+        "aarch64": "arm64",
+        "x86_64": "x64",
+    }
+
+    angle_x64 = arch[target.split('-')[0]]
+    print("angle_x64 Architecture = {}".format(angle_x64))
+    angle_default_path = path.join(os.getcwd(),
+                                   "support\\hololens\\packages\\ANGLE.WindowsStore.Servo.2.1.13\\bin\\UAP\\")
+    angle_default_path += angle_x64
+
+    # Nuget executable command
+    nuget_exe = path.join(os.getcwd(), ".servo\\msvc-dependencies\\nuget\\08-08-2019\\nuget.exe ")
```

Comment:
We should use `path.join(self.msvc_package_dir('nuget'), 'nuget.exe')` instead of hardcoding this (https://github.com/servo/servo/blob/91469aa47215c8e1b4e4ae5643ca793b3f65af9b/python/servo/command_base.py#L579-L580)

Reply:
Done.